src/openms/include/OpenMS/KERNEL/ChromatogramPeak.h

```diff
@@ -85,6 +85,12 @@ namespace OpenMS
       intensity_(p.intensity_)
     {}
 
+    /// Constructor with position and intensity
+    inline ChromatogramPeak(const PositionType position, const IntensityType intensity) :
```

Comment:
just for clarity, I would rename the argument from "position" to "retention_time"

Reply:
Sure!
About the pxd file, is it necessary to give argument names? Because for the rest of the file they are not present @hroest

Reply:
no its not necessary but it helps people to understand the code :-)